source/write/replace.txt

```diff
@@ -0,0 +1,210 @@
+of a document in which the ``name`` field value is ``'Pizza Town'``. It then prints
+the number of modified documents:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/write/replace.php
+      :start-after: start-replace-one
+      :end-before: end-replace-one
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      Modified documents: 1
+
+.. important::
+
+   The values of ``_id`` fields are immutable. If your replacement document specifies 
+   a value for the ``_id`` field, it must match the ``_id`` value of the existing document.
+
+Modify the Replace Operation
+----------------------------
+
+You can modify the behavior of the ``MongoDB\Collection::replaceOne()`` method
+by passing an array that specifies option values as a parameter. The following
+table describes some options you can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``upsert``
+     - | Specifies whether the replace operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the {+mdb-server+} manual.
+       | Defaults to ``false``.
+
+   * - ``bypassDocumentValidation``
+     - | Specifies whether the replace operation bypasses document validation. This lets you 
+         replace documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Gets or sets the index to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the {+mdb-server+} manual.
+
+   * - ``session``
+     - | Specifies the client session to associate with the operation.
+
+   * - ``let``
+     - | Specifies a document with a list of values to improve operation readability.
+         Values must be constant or closed expressions that don't reference document
+         fields. For more information, see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Attaches a comment to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual.
```

Comment:
Did you omit the options on purpose: `codec`, `session`, `writeConcern`?

Reply:
For methods that have a long list of options, we usually don't include them all (and mention that the table only "describes some options"). Let me know if you think any of those omitted options should be included-  `upsert` was also left out.

Reply:
No problem, if that's the usage.